src/Storage/Storage.Test/LiveTests/TestLiveScenarios.ps1

```diff
@@ -0,0 +1,40 @@
+Invoke-LiveTestScenario -Name "Creates a Storage account" -Description "Test create storage account" -ScenarioScript `
+{
+    param ($rg)
+
+    $rgName = $rg.ResourceGroupName
+    $name = New-LiveTestResourceName
+    $vaultLocation = "westus"
```

Comment:
Please rename the $vaultLocation to $location and update below invocation accordingly.